src/Workspaces/Remote/ServiceHub/Services/ProcessTelemetry/RemoteProcessTelemetryService.PerformanceReporter.cs

```diff
@@ -38,12 +38,11 @@ private class PerformanceReporter : GlobalOperationAwareIdleProcessor
                 TelemetrySession telemetrySession,
                 IPerformanceTrackerService diagnosticAnalyzerPerformanceTracker,
                 IGlobalOperationNotificationService globalOperationNotificationService,
-                TimeSpan reportingInterval,
                 CancellationToken shutdownToken)
                 : base(
                     AsynchronousOperationListenerProvider.NullListener,
                     globalOperationNotificationService,
-                    reportingInterval,
+                    backOffTimeSpan: TimeSpan.FromMinutes(2),
```

Comment:
❔ was this ever passed a different value in tests?

Reply:
PerformanceReporter is not explicitly tested.